client/wp-admin-scripts/payment-method-promotions/payment-promotion-row.tsx

```diff
@@ -12,74 +12,84 @@ import {
 } from '@woocommerce/data';
 import { recordEvent } from '@woocommerce/tracks';
 import { useDispatch, useSelect } from '@wordpress/data';
+import { sanitize } from 'dompurify';
 import { __ } from '@wordpress/i18n';
 import _ from 'lodash';
 
 /**
  * Internal dependencies
  */
-import './wc-payments-row.scss';
+import './payment-promotion-row.scss';
 
-type WCPaymentsRowProps = {
+function sanitizeHTML( html: string ) {
+	return {
+		__html: sanitize( html, {
+			ALLOWED_TAGS: [ 'a', 'img' ],
+			ALLOWED_ATTR: [ 'href', 'src', 'class', 'alt', 'target' ],
+		} ),
+	};
+}
+
+type PaymentPromotionRowProps = {
+	pluginSlug: string;
 	sortColumnContent: string;
 	descriptionColumnContent: string;
+	titleLink: string;
+	title: string;
 	subTitleContent?: string;
 };
 
-const WC_PAY_SLUG = 'woocommerce-payments';
-export const WCPaymentsRow: React.FC< WCPaymentsRowProps > = ( {
+export const PaymentPromotionRow: React.FC< PaymentPromotionRowProps > = ( {
+	pluginSlug,
 	sortColumnContent,
 	descriptionColumnContent,
+	title,
+	titleLink,
 	subTitleContent,
 } ) => {
 	const [ installing, setInstalling ] = useState( false );
 	const { installAndActivatePlugins }: PluginsStoreActions = useDispatch(
 		PLUGINS_STORE_NAME
 	);
 	const { createNotice } = useDispatch( 'core/notices' );
-	const wcPayInstallationInfo = useSelect( ( select: WCDataSelector ) => {
+	const installationInfo = useSelect( ( select: WCDataSelector ) => {
 		const { getPaymentGateway } = select( PAYMENT_GATEWAYS_STORE_NAME );
 		const activePlugins: string[] = select(
 			PLUGINS_STORE_NAME
 		).getActivePlugins();
-		const isWCPayActive =
-			activePlugins && activePlugins.includes( WC_PAY_SLUG );
-		let wcPayGateway;
-		if ( isWCPayActive ) {
-			wcPayGateway = getPaymentGateway(
-				WC_PAY_SLUG.replace( /\-/g, '_' )
+		const isActive = activePlugins && activePlugins.includes( pluginSlug );
+		let paymentGateway;
+		if ( isActive ) {
+			paymentGateway = getPaymentGateway(
+				pluginSlug.replace( /\-/g, '_' )
 			);
 		}
 
 		return {
-			isWCPayActive,
-			wcPayGateway,
+			isActive,
+			paymentGateway,
 		};
 	} );
 
 	useEffect( () => {
 		if (
-			wcPayInstallationInfo.isWCPayActive &&
-			wcPayInstallationInfo.wcPayGateway &&
-			wcPayInstallationInfo.wcPayGateway.settings_url
+			installationInfo.isActive &&
```

Comment:
Might be nice to destructure these above coming out of the `useSelect` for readability.